basics/Main.java

```diff
@@ -0,0 +1,57 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Random;
+public class Main {
+  public static void main (String[] args){
+    int dogCount = 1;
+    int catCount = 2;
+    int mouseCount = 3;
+    System.out.println("i have " + dogCount + " " + pluralize("dog", dogCount));
+    System.out.println("i have " + catCount + " " + pluralize("cat", catCount));
+    System.out.println("i have " + mouseCount + " " + pluralize("mouse", mouseCount));
+    System.out.println(flipNHeads(5));
+    clock();
+  }
+
+  public static String pluralize(String word, int num){
+    if(num == 0 || num > 1){
+      return word + 's';
+    }else {
+      return word;
+    }
+  }
+
+  public static String flipNHeads(int n){
+    int cnt = 0;
```

Comment:
I think a more descriptive variable name would make this a little easier for other devs to read. Without reading further I wouldn't know you're counting the number of times heads is flipped consecutively